Cargo.toml

```diff
@@ -30,6 +30,11 @@ std = [
 derive = [
     "scale-info-derive"
 ]
+# Derive `TypeInfo` for our own types.
+dogfood = [
+    "derive"
+]
+# Enables decoding and deserialization of portable scale-info type metadata.
```

Comment:
Does this comment belong on top of the feature?

Reply:
No, I know where this comes from, it's the comment for the "full" feature, with the leading `E` capitalized. Removing.